### PR TITLE
Added new points for office 365 pro plus

### PR DESCRIPTION
**Pull request by RAJU2529**
According to user issue, I added two points and Highlighted
**https://github.com/MicrosoftDocs/OfficeDocs-DeployOffice/issues/481**

Office 365 ProPlus is supported on user-dedicated virtual desktop infrastructure (VDI).
Office 365 ProPlus only supports Remote Desktop Services (RDS) when purchased through a Volume Licensing Program.

added relative topics link

### CODE REVIEW

File: DeployOffice/deploy-office-365-proplus-by-using-remote-desktop-services.md [hunk 1]
```diff
@@ -15,6 +15,9 @@ description: "If you use Remote Desktop Services (RDS) to provide shared compute
 
 If you use Remote Desktop Services (RDS) to provide shared computers to users in your organization, you can install Office 365 ProPlus on those computers. But, you have to use the Office Deployment Tool and enable [shared computer activation](overview-of-shared-computer-activation-for-office-365-proplus.md) to do the installation.
 
+## Office 365 ProPlus is supported on user-dedicated virtual desktop infrastructure (VDI).
```
Comment (ItzLevvie) [inline DeployOffice/deploy-office-365-proplus-by-using-remote-desktop-services.md]: Please make this a note by doing the something along the lines of:
```
> [!NOTE]
> Office 365 ProPlus is supported on user-dedicated virtual desktop infrastructure (VDI) and only supports Remote Desktop Services (RDS) when purchased through a Volume Licensing Program.
```

File: DeployOffice/deploy-office-365-proplus-by-using-remote-desktop-services.md [hunk 2]
```diff
@@ -119,4 +122,6 @@ After you've completed all the RDS configuration steps, users can connect to any
 
 [Remote Desktop Services Overview (Windows Server 2012 and Windows Server 2012 R2)](https://go.microsoft.com/fwlink/p/?LinkID=268149)
 
+(https://docs.microsoft.com/en-gb/office365/servicedescriptions/office-applications-service-description/office-applications-service-description)
```
Comment (ItzLevvie) [inline DeployOffice/deploy-office-365-proplus-by-using-remote-desktop-services.md]:
```suggestion
[Office Applications Service Description](https://docs.microsoft.com/office365/servicedescriptions/office-applications-service-description/office-applications-service-description)
```

File: DeployOffice/deploy-office-365-proplus-by-using-remote-desktop-services.md (hunk 1 above)
Comment (RAJU2529) [reply DeployOffice/deploy-office-365-proplus-by-using-remote-desktop-services.md]: Thanks for reviewing, I will edit now

File: DeployOffice/deploy-office-365-proplus-by-using-remote-desktop-services.md [hunk 3]
```diff
@@ -15,6 +15,9 @@ description: "If you use Remote Desktop Services (RDS) to provide shared compute
 
 If you use Remote Desktop Services (RDS) to provide shared computers to users in your organization, you can install Office 365 ProPlus on those computers. But, you have to use the Office Deployment Tool and enable [shared computer activation](overview-of-shared-computer-activation-for-office-365-proplus.md) to do the installation.
 
+## Office 365 ProPlus is supported on user-dedicated virtual desktop infrastructure (VDI).
+## Office 365 ProPlus only supports Remote Desktop Services (RDS) when purchased through a Volume Licensing Program.
```
Comment (ItzLevvie) [inline DeployOffice/deploy-office-365-proplus-by-using-remote-desktop-services.md]: This line can be removed; see https://github.com/MicrosoftDocs/OfficeDocs-DeployOffice/pull/483/files#r344241601

Comment (RAJU2529) [reply DeployOffice/deploy-office-365-proplus-by-using-remote-desktop-services.md]: ok sir. From myside, now what I do Sir.

**Comment by RAJU2529** [comment]
@DHB-MSFT. Any updates ?

**Comment by ItzLevvie** [comment]
@RAJU2529 You've not added the suggested changes that I have made above yet.

**Comment by RAJU2529** [comment]
@ItzLevvie. Sir just now I removed the sentence from line 19.
and tell me  next what should I do

**Comment by ItzLevvie** [comment]
cc @RAJU2529 check the PR on your repo

**Comment by ItzLevvie** [comment]
cc @RAJU2529

**Comment by RAJU2529** [comment]
@illfated. This  PR  conflicts, how i resolve it, when i click on resolve conflicts, it redirects o page, showing red bar at the bottom. 
I dont know what to do

**Comment by illfated** [comment]
@RAJU2529 : Thank you for asking. It appears that this PR has been sitting here without being followed up for a long time.

I recommend closing this PR and opening a new one, starting with a fresh and up-to-date file.
(it is not like it will be much of a delay in the follow-up process so far, anyway)

**Comment by DHB-MSFT** [comment]
@RAJU2529 -- I'll try to look at this problem this week. Might be because there have been other changes to the article since you submitted this pull request. Thanks.

**Comment by illfated** [comment]
When starting a new PR, make sure you browse to the newest version of the same file, then start editing the file.

**Comment by DHB-MSFT** [comment]
Closing. Old request and article recently updated. If you want to recommend changes, create a new pull request based on the most recent version of the article. Thank you.

**Pull request closed without merging**